modules/electrophysiology_browser/ajax/get_electrophysiology_session_data.php

```diff
@@ -0,0 +1,327 @@
+<?php declare(strict_types=1);
+/**
+ * Get Electrophysiology Session Data.
+ *
+ * This retrieves the entire electrophysiology session data
+ * for a candidate to be visible on the electrophysiology_session form.
+ *
+ * PHP Version 7
+ *
+ * @category LORIS
+ * @package  Electrophysiology_Session
+ * @author   Muhammad Khan <muhammad.khan@mcin.ca>
+ * @license  LORIS license
+ * @link     https://github.com/aces/Loris-Trunk
+ */
+
+require_once 'ElectrophysioFile.class.inc';
+
+/**
+ * User permission verification:
+ * Only users that have the electrophysiology view all sites permission OR
+ * have the view site permission and belong to candidates session site
+ * are able to fetch the electrophysiology session data.
+ */
+$user      = \NDB_Factory::singleton()->user();
+$timePoint = \NDB_Factory::singleton()->timepoint(
+    intval($_REQUEST['sessionID'])
```

Comment:
Could you update this file to use sessionID consistently? It's being used as both an int and a string in this file.

Linking #4369 because it would help in a case like this.

Reply:
done in the next commit